run-checker/run-checker.sh

```diff
@@ -46,7 +46,13 @@ no-tls1_2-method no-dtls1-method no-dtls1_2-method no-siphash no-tls1_3 no-sm2
 no-sm3 no-sm4 enable-trace no-legacy no-cached-fetch no-autoload-config
 'no-buildtest-c++' no-bulk no-cmp no-ktls no-module no-padlockeng
 no-pinshared no-secure-memory no-siv no-uplink enable-acvp-tests enable-fips
-'enable-fips no-fips-securitychecks'
+'enable-fips no-fips-securitychecks' 'enable-fips enable-acvp-tests'
+'enable-fips no-cmac' 'enable-fips no-cmac' 'enable-fips no-des'
```

Comment:
On second thoughts are the FIPS + disabled algorithms actually meaningful?   It seems unlikely that the FIPS provider could be built in an approved manner if they are excluded.

Reply:
Well, somebody could do a private validation with FIPS and disabled algorithms. But I am not sure we need to worry about such in runchecker builds.

Reply:
I've pulled the FIPS + disabled algorithms options.